### DISCUSSION TITLE
Allow Pause on Task instance level

### DISCUSSION THREAD

**Discussion by Pad71** [apache/airflow#38181] [General]
### Description

Currently, the Airflow UI will allow you to pause the Dag.
Airflow UI should also allow to use similar functionality for the  task instance. 
The Functionality similar to the Dag Level Pause - but this time on the task instance level.

### Use case/motivation

Enable to solve use cases when it is necessary to suspend running at the task level

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by Taragolis** [comment]
This one required significant changes in Airflow.
It should comes thought AIP (Airflow Improvement Proposals) first, see: https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals
